test/interpreter/function-roman.spec.ts

```diff
@@ -0,0 +1,82 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {ErrorMessage} from '../../src/error-message'
+import {adr, detailedError} from '../testUtils'
+
+describe('Function ROMAN', () => {
+  it('should return #NA! error with the wrong number of arguments', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=ROMAN()', '=ROMAN(1, 1, 1)'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+    expect(engine.getCellValue(adr('B1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should properly truncate values and use defaults', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=ROMAN(499)'],
+      ['=ROMAN(499, TRUE())'],
+      ['=ROMAN(499, FALSE())'],
+      ['=ROMAN(499.9)'],
+      ['=ROMAN(499, 1.1)'],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toEqual('CDXCIX')
+    expect(engine.getCellValue(adr('A2'))).toEqual('ID')
```

Comment:
I have different results in Excel. According to the documentation, both `default` aka mode `0` is "Classic", the same as `TRUE`. 

`FALSE` should be the same as mode `4`. Maybe we can add here an example with mode 4 "Simplified", next to the `FALSE`.
![Screenshot 2020-10-22 at 19 40 11](https://user-images.githubusercontent.com/4718998/96909558-7a0d1c80-149e-11eb-828b-b0098c6b9fbe.png)


Source: https://support.microsoft.com/en-us/office/roman-function-d6b0b99e-de46-4704-a518-b45a0f8b56f5

Reply:
ok, thanks!